recipes/ninja/all/conanfile.py

```diff
@@ -1,13 +1,15 @@
-from conan import ConanFile
+from conan import ConanFile, conan_version
 from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 from conan.tools.files import copy, get
+from conan.tools.scm import Version
 import os
 
 required_conan_version = ">=1.46.0"
```

Comment:
```suggestion
required_conan_version = ">=1.52.0"
```

due to `conan_version`

Reply:
Happy to bump the required Conan version as that's what I had in my original change but I had relaxed it to minimize the diff.